openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -11,10 +17,63 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
+from openedx_events.learning import signals  # pylint: disable=reimported
+from openedx_events.tests.utils import FreezeSignalCacheMixin
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# If a signal is explicitly not for use with the event bus, add it to this list
```

Comment:
[blocker] 
1. Can you add an additional note like the following
```suggestion
# If a signal is explicitly not for use with the event bus, add it to this list
#   and document why in the event's annotations.
```

2. Also, could you add that documentation to the discussions event you included?

Reply:
@whuang1202: This still has not been addressed.

Reply:
oh mb I only committed all the files I changed under the avro folder, let me make a commit real quick

Reply:
Thanks. I see the change. Note: from a commit message perspective, you may want to squash this with a "docs:" commit message and the rest in a "test:" commit. However, if this gets too complicated for you, a single commit with a "test:" commit message will be fine.